edb/schema/pointers.py

```diff
@@ -874,6 +874,22 @@ def compile_expr_field(self, schema, context, field, value):
         else:
             return super().compile_expr_field(schema, context, field, value)
 
+    def _apply_field_ast(self, schema, context, node, op):
+        if context.descriptive_mode:
+            # When generating AST for DESCRIBE AS TEXT, we want to
+            # omit 'readonly' flag if it's inherited and it actually
+            # has the default value.
+            field_name = 'readonly'
```

Comment:
```suggestion
```

Reply:
I've updated this code. Also rebased on master and resolved the coverage percentage conflict.